doc/source/science_guide/sg_dynamics.rst

```diff
@@ -395,28 +386,43 @@ Viscous-Plastic
 The VP constitutive law is given by
 
 .. math::
-   \sigma_{ij} = 2 \eta \dot{\epsilon}_{ij} + (\zeta - \eta) D_D - P_R(1 - k_t)\frac{\delta_{ij}}{2}
+   \sigma_{ij} = 2 \eta \dot{\epsilon}_{ij} + (\zeta - \eta) D_D - P_R\frac{\delta_{ij}}{2}
    :label: vp-const
 
-where :math:`\eta` and :math:`\zeta` are the bulk and shear viscosities.
+where :math:`\eta` and :math:`\zeta` are the bulk and shear viscosities and
+:math:`P_R` is a “replacement pressure” (see :cite:`Geiger98`, for example),
+which serves to prevent residual ice motion due to spatial
+variations of the ice strength :math:`P` when the strain rates are exactly zero.
+
 An elliptical yield curve is used, with the viscosities given by
 
 .. math::
    \zeta = {P(1+k_t)\over 2\Delta},
 
 .. math::
-   \eta  = e^{-2} \zeta,
+   \eta  = e_g^{-2} \zeta,
 
 where
 
 .. math::
-   \Delta = \left[D_D^2 + {1\over e^2}\left(D_T^2 + D_S^2\right)\right]^{1/2}
+   \Delta = \left[D_D^2 + {e_f^2\over e_g^4}\left(D_T^2 + D_S^2\right)\right]^{1/2}.
+
+The ice strength :math:`P` is a function of the ice thickness distribution as
+described in the `Icepack Documentation <https://cice-consortium-icepack.readthedocs.io/en/master/science_guide/index.html>`_.
+   
+Two modifications to the standard VP rheology of :cite:`Hibler79` are available.
+First, following the approach of :cite:`Konig10` (see also :cite:`Lemieux16`), the
+elliptical yield curve can be modified such that the ice has isotropic tensile strength.
+The tensile strength :math:`T_p` is expressed as a fraction of :math:`P`, that is :math:`T_p=k_t P`
+where :math:`k_t` should be set to a value between 0 and 1 (this can
+be changed at runtime with the namelist parameter ``Ktens``).
```

Comment:
It would be clearer if `T_p` appeared in the equations below, or at least mention how its effect is incorporated in the viscous coefficients (since `k_t` is removed from the equations).

Reply:
I have removed T_p in the doc.